modules/appeals_api/spec/requests/schemas/shared_schemas_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require AppealsApi::Engine.root.join('spec', 'spec_helper.rb')
+
+describe AppealsApi::Schemas::SharedSchemasController, type: :request do
+  include FixtureHelpers
+
+  def base_path(appeal_type, schema_type)
+    "/services/appeals/#{appeal_type}/v2/schemas/#{schema_type}"
+  end
+
+  describe '#show' do
+    describe "schema type 'non_blank_string'" do
+      appeal_types = %w[notice_of_disagreements higher_level_reviews supplemental_claims]
```

Comment:
Lots of repetition in these tests - not a huge deal but it might be nice to simplify this into a local shared example or somesuch...

Reply:
Cool i'll get a ticket going